backend/src/main/java/gov/cdc/usds/simplereport/api/DefaultArgumentValidation.java

```diff
@@ -0,0 +1,112 @@
+package gov.cdc.usds.simplereport.api;
+
+import graphql.GraphQLError;
+import graphql.GraphqlErrorBuilder;
+import graphql.execution.ResultPath;
+import graphql.language.SourceLocation;
+import graphql.schema.GraphQLArgument;
+import graphql.schema.GraphQLDirective;
+import graphql.schema.GraphQLFieldDefinition;
+import graphql.schema.GraphQLFieldsContainer;
+import graphql.schema.GraphQLInputObjectType;
+import graphql.schema.GraphQLList;
+import graphql.schema.GraphQLModifiedType;
+import graphql.schema.GraphQLScalarType;
+import graphql.schema.GraphQLType;
+import graphql.validation.rules.ValidationEnvironment;
+import graphql.validation.rules.ValidationRule;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+public class DefaultArgumentValidation implements ValidationRule {
+  private static final Set<String> DIRECTIVES_OVERRIDING_DEFAULT_RULE = Set.of("Size", "Pattern");
```

Comment:
Would be nice to have a little explanation of what magic these are for and whether one should ever change them.